Samsung/A32-SystemUI/res/values/dimens.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item type="dimen" name="config_screenBrightnessDimFloat">0.047058824</item>
```

Comment:
that's not a sysui prop...

Reply:
im sorry i wasnt aware. i am not an expert with overlays. i'll remove them

Reply:
configs removed